$NetBSD: patch-ad,v 1.2 2014/11/02 13:31:11 spz Exp $

Add ability to parse unsigned integers as externalized by NetBSD proplib(3).

--- Lib/plistlib.py.orig	2014-06-30 02:05:30.000000000 +0000
+++ Lib/plistlib.py
@@ -463,7 +463,7 @@ class PlistParser:
     def end_false(self):
         self.addObject(False)
     def end_integer(self):
-        self.addObject(int(self.getData()))
+        self.addObject(int(self.getData(), 0))
     def end_real(self):
         self.addObject(float(self.getData()))
     def end_string(self):
